src/v2/stack_aggregator.py

```diff
@@ -144,16 +147,46 @@ def __init__(self,
         self._normalized_packages = normalized_packages
         self._normalized_package_details = None
         self._result = None
+        self.filtered_vul = {}
 
```

Comment:
Even initiate_unknown_package_ingestion can be made as a method in Aggregator class and override in GoAggregator to keep empty to imply there is no unknown handling for golang.

Reply:
Ack